pkgs/development/python-modules/range-typed-integers/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytestCheckHook }:
+
+buildPythonPackage rec {
+  pname = "range-typed-integers";
+  version = "1.0.1";
```

Only Python > 3.8 is [supported](https://github.com/theCapypara/range-typed-integers/blob/1.0.1/pyproject.toml#L6).

```suggestion
  version = "1.0.1";
  format = "pyproject";

  disabled = pythonOlder "3.8";
```